Shared/Samples/Animate images with image overlay/AnimateImagesWithImageOverlayView.swift

```diff
@@ -0,0 +1,174 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AnimateImagesWithImageOverlayView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The text for the start stop button.
+    @State private var startStopButtonText = "Start"
+    
+    var body: some View {
+        VStack {
+            // Create a scene view to display the scene.
+            SceneView(scene: model.scene, imageOverlays: [model.imageOverlay])
+                .onAppear {
+                    // Create display link and load first image.
+                    model.displayLink = model.makeDisplayLink()
```

Comment:
The thought was that the display link should be stopped/invalidated when the view that uses it disappears, so no invisible frame is rendered to waste computation. But since this sample only makes the view disappear when on exit, I'm OK to move them to init/deinit.